src/form/ColorMapInput.tsx

```diff
@@ -270,129 +270,126 @@ const ColorMapInput: React.FC<ColorMapProps & InjectedIntlProps> = (props) => {
   const placeholderMaximumColorRange = intl.formatMessage({ id: "placeholder_maximum_color_range" })
 
   return (
-    <label
-      htmlFor={name}
```

Comment:
I think a simpler solution is to remove the `htmlFor={name}` attribute of the label here instead of removing the whole label tag (it has some specific styling for the label namely `marginBottom` which is now totally removed)

Reply:
removing the htmlFor={name} did not resolve the issue on firefox. Clicking anywhere on the colormap input element still opened the dropdown. Didnot test for chrome.
I will try to fix the issue with marginBottom

Reply:
Strange. It worked on my firefox. Could you try to give it a different name? such as `htmlFor={name+'colorMapInput'}`. The problem is there is a conflict here. The `SelectBox` also has a lable with `htmlFor={name}`, so if they don't have the same name then it would not trigger the dropdown under the color map input.